roseau/load_flow_single/network.py

```diff
@@ -993,15 +993,13 @@ def _propagate_voltages(self) -> None:
 
     def _get_starting_voltage(self) -> tuple[complex, VoltageSource]:
```

Comment:
Simplified as the old logic was needed with multiphase networks only. Here there is no need to create a sorted list.